node/accounts-bot/index.js

```diff
@@ -0,0 +1,183 @@
+const {google} = require('googleapis');
+const SHEETS_SCOPE = 'https://www.googleapis.com/auth/spreadsheets.readonly';
+const ACCOUNTS_SHEET_ID = '1kxW15ZI48mh4KkvgsMpg7gInmEQmyKYRnZdbUOSMRnU';
+const ACCOUNTS_SHEET_URL = 'https://docs.google.com/spreadsheets/d/1kxW15ZI48mh4KkvgsMpg7gInmEQmyKYRnZdbUOSMRnU/edit#gid=0';
+const ACCOUNT_IMAGE_URL = 'https://www.gstatic.com/images/icons/material/system_gm/1x/account_circle_black_18dp.png';
+
+/**
+ * Looks up the account owner for the company requested.
+ *
+ * @param {!express:Request} req HTTP request context.
+ * @param {!express:Response} res HTTP response context.
+*/
+exports.accountsBot = async (req, res) => {
+  const message = req.query.message || req.body.message;
+  if (!message || !message.text) {
+    console.log('invalid input');
+    res.send(createTextResponse('Please provide an account to look up'));
+  }
+
+  const accountName = getAccountName(JSON.stringify(message.text));
```

Comment:
Why JSON.stringify here? message.text should just be a string already.

Reply:
Oh, and it would simplify getAccountName as it wouldn't have the extra quotes in the string.

Reply:
#getAccountName is removed